api/lib/source/polkadotV0-source.js

```diff
@@ -245,9 +245,9 @@ class polkadotAPI {
     const { free, reserved, feeFrozen } = account.data.toJSON()
     const totalBalance = BigNumber(free).plus(BigNumber(reserved))
     const freeBalance = BigNumber(free).minus(feeFrozen)
-    const stakedBalance = BigNumber(
+    const stakedBalance = stakingLedger ? BigNumber(
```

Comment:
ah cool. Didn't know this one could be undefined, thought it would always return some Object. Makes sense

Reply:
Implementing exactly the same rn ;-), closing https://github.com/luniehq/lunie/issues/4936